src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -0,0 +1,227 @@
+
+## Configure the message template [#configure-jira-message-template]
+
+Upon successful connection, Jira fields are fetched from your account and automatically mapped to your Jira instance. 
+
+To help you get started, recommended fields and values are automatically presented. You can add or remove fields and change the values to customize the notification to your needs.
+
+![A screenshot of the Jira-specific custom variables menu.](./images/jira-custom-variables.png "The Jira-specific custom variables menu.")
+
+<figcaption>The variables menu changes depending on the destination integration you're using.</figcaption>
+
+## Send a test notification [#send-test-notification]
+
+Users can see how the JIRA issue will appear by clicking a test notification with default field values. If successful, a Jira ticket is created and a link to that ticket appears. 
+  
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="servicenow"
+    title="ServiceNow"
+  >
+  Integrate New Relic with ServiceNow and automatically create and update incidents.
+
+## Roles [#roles]
+
+For this integration to work, your ServiceNow user details need read permissions for the tables: `sys_dictionary`, `sys_choice`, `sys_user`, and `change_request` (optional).
+
+You need a ServiceNow admin role enable two-way integration between ServiceNow and New Relic One.
+
+## Set up a destination [#set-up-service-destination]
+
+To create a ServiceNow destination, enter the following information:
+
+* **Name:** custom name to identify the destination
+* **Domain:** the URL of the destination
+* **User-name:**  the name of the user
+* **Password:**  the user name’s password 
+
+## Two-way Sync [#two-way-servicenow]
+
+You can configure a two-way integration with ServiceNow incidents so that when state updates for the incident (resolved or closed), it triggers an update in the corresponding New Relic issue state. 
+
+Here are some required steps to remember when configuring the two-way integration:
+
+1. Turn on the `two-way integration` toggle. 
+
+2. Open and download this [XML file](), which includes the business rule triggering events back to New Relic One.
+
+3. In the ServiceNow sidebar menu, go to **System Definition > Business Rules**.
+
+4. Click the menu icon in one of the column headers, select **Import XML** and upload the XML file you downloaded.
+
+Before saving the destination, we recommend testing the connection by clicking the **test connection** button.
+
+## Configure the message template [#configure-servicenow-message-template]
+
+Upon a successful connection, ServiceNow incident table columns are fetched from your account and automatically mapped to your ServiceNow instance. 
+
+To help you get started, recommended fields and values are automatically presented.
```

Comment:
same as previous similar statement. It's passive voice, but active is a little tricky in this case. The figure caption handles this nicely.